terraform/modules/pre-auth-lambda/lambda/lambda_function.py

```diff
@@ -0,0 +1,76 @@
+import os
+import datetime
+import dateutil
+from dateutil.relativedelta import relativedelta
+import boto3
+from boto3.dynamodb.conditions import Key
+
+tableName = os.environ.get("TABLE_NAME")
+regionName = os.environ.get("AWS_REGION")
+
+dynamodb = boto3.resource("dynamodb", region_name=regionName)
+table = dynamodb.Table(tableName)
+
+def queryUser(username):
```

Comment:
2 blank lines required before each function

Reply:
This is done.